script/migration/seeds/TrackSeeder.php

```diff
@@ -12,34 +13,101 @@ class TrackSeeder extends AbstractSeed
 
     public function run()
     {
-
-        if (!$this->hasTable('track')) {
-            return;
-        }
+        $this->query("UPDATE track_file SET version_id = NULL");
+        $this->query("UPDATE version SET file_id = NULL");
+        $this->query("DELETE FROM point");
+        $this->query("DELETE FROM version");
+        $this->query("DELETE FROM track_file");
+        $this->query("DELETE FROM optimized_point");
+        $this->query("DELETE FROM track");
 
         $track = $this->table('track');
 
-        /* Disable constrains check during truncate */
-        $this->execute('SET FOREIGN_KEY_CHECKS = 0; ');
-        $track->truncate();
-        $this->execute('SET FOREIGN_KEY_CHECKS = 1; ');
-
-        $count = rand(1, 5);
-        for ($i = 0; $i < $count; $i++) {
+        for ($i = 0; $i < 15; $i ++) {
+            $trackId = uniqid();
             $data = [
-                'id' => uniqid(),
-                'name' => 'Dummy track '.$i,
-                'last_check' => date('Y-m-d H:i:s', strtotime(sprintf("+%d hours", $i))),
+                'id' => $trackId,
+                'last_check' => date('Y-m-d H:i:s', strtotime(sprintf("-%d hours", $i))),
+                'name' => 'Dummy track ' . $i,
                 'point_north_east_lat' => 0,
                 'point_north_east_lng' => 0,
                 'point_south_west_lat' => 0,
                 'point_south_west_lng' => 0,
-                'type' => $this::TYPE_CYCLING,
-                'created_at' => date('Y-m-d H:i:s', strtotime(sprintf("+%d hours", $i))),
-                'visibility' => $this::VISIBILITY_PUBLIC,
+                'type' => self::TYPE_CYCLING,
```

Comment:
Generate also hiking tracks?
`'type' => ($i % 2) ? self::TYPE_CYCLING : self::TYPE_HIKING`

Maybe increase count to 30?

Reply:
Added logic for TrackVisibility & TrackType